src/angularjs/src/app/places/list/place-list.html

```diff
@@ -37,31 +37,26 @@ <h2 class="sidebar-title">City/Town Search</h2>
 
                 <!-- Compare dropdown -->
                 <div class="form-group">
-                    <label for="3">Compare</label>
-                    <div class="dropdown">
-                      <button class="btn btn-default dropdown-toggle btn-block" type="button" id="dropdownMenu1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                        3 Selected
-                        <span class="caret"></span>
-                      </button>
-
-                      <!-- We are using bootstraps dropdown js -->
-                      <ul class="dropdown-menu compare-dropdown" aria-labelledby="dropdownMenu1">
-
-                        <!-- Only up to 3 to compare. Duplicate li and children for each item -->
-                        <li>
-                            <div class="compare-title">Los Angeles, California</div>
-                            <div class="network-score small">90</div>
-                            <a href="#" class="compare-remove">Remove</a>
-                        </li>
-                        <li>
-                            <div class="compare-title">Los Angeles, California</div>
-                            <div class="network-score small">90</div>
+                    <label for="compare-dropdown">Compare</label>
+                    <select ng-model="placeList.placeToRemoveFromComparison" name="compare-dropdown"
+                        ng-disabled="!placeList.comparePlacesCount()" class="form-control"
+                        id="role" type="text">
+                    <option selected disabled value="">{{placeList.comparePlacesCount()}} Selected</option>
+                    <option ng-repeat="neighborhood in placeList.comparePlaces | filter:{uuid: '!!'}"
```

Comment:
Right, it gets lost; that's why I was going to make a separate issue for the styling. Prototype used plain Bootstrap, but we have Angular components here that apparently have their own ideas about styling. Good find on the `compare-dropdown` class.